charts/sourcegraph/templates/qdrant/qdrant.ConfigMap.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.qdrant.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.qdrant.name }}
+  labels:
+    deploy: sourcegraph
+    app.kubernetes.io/component: qdrant
+data:
+  config.yaml: |
+    debug: true
+    log_level: INFO
+    storage:
+      storage_path: /data
+      snapshots_path: /data/storage
+      on_disk_payload: true
```

Comment:
these seem to be global flags; we should parameterize them

Reply:
Do we need to parameterize them even if we don't want them to be configurable? We control the volume mount location as well, so I can't think of any reason to move the storage path elsewhere

Reply:
at least `debug` should be configurable, no?

I'm not sure the implication of `debug: true` means for qdrant, but we can only expose some of the fields here.

Reply:
Oooh, gotcha, yeah definitely. No idea why my eyes focused on just the storage path 😂